src/lib/classes/lsp3-universal-profile.spec.ts

```diff
@@ -289,6 +294,75 @@ describe('LSP3UniversalProfile', () => {
     });
   });
 
+  describe('Deploying UP with specified bytecode', () => {
+    describe('Deploy ERC725Accout from custom bytecode', () => {
+      let deployedContracts: DeployedContracts;
+      it('should not deploy UP base contract', async () => {
```

Comment:
Same here, maybe you can add the assertion in the test to check if the bytecode of your deployed contract match with the bytecode you have passed to the `deploymentOptions` object?

Reply:
This would be good but doesn't work because of what we discussed last week regarding the constructor logic not being included at the deployed address. This means what is passed and what is deployed is different so cant be compared